lib/tasks/after_tasks.rake

```diff
@@ -618,7 +618,7 @@ namespace :After do
   end
   
   desc "Reveal works and creators hidden upon invitation to unrevealed or anonymous collection"
-  task(:unhide_invited_works) do
+  task(unhide_invited_works: :environment) do
     works = Work.where("in_anon_collection IS true OR in_unrevealed_collection IS true")
     works.find_in_batches.each do |batch|
       batch.each do |work|
```

Comment:
Should we print the total `works` at the start, and `print(".") && STDOUT.flush` after each batch, similar to the two most recent tasks?

Reply:
Loading production environment (Rails 5.1.6.2)
2.3.4 :001 > Work.where("in_anon_collection IS true OR in_unrevealed_collection IS true").count
 => 15279